### ISSUE TITLE
Using only one AZ and Unresolved resource dependencies error

### ISSUE THREAD

**Issue by porschberg** [smoketurner/serverless-vpc-plugin#243]
Hi,
I try to to create the stack with only ONE availability zone. 

In my serverless.yml I do something like:

```
  vpcConfig:
    createNatGateway: true
    zones:
      - eu-central-1b

```
But I get an error when I deploy -->

`Error: The CloudFormation template is invalid: Unresolved resource dependencies [RDSSubnetGroup] in the Outputs block of the template
`
All is okay when I omit the zone entry. In this case three external IPs are created. (But I want only one.)

What is the problem?

**Comment by ardalann** [comment]
I'm experiencing the exact same issue.

**Comment by jplock** [comment]
I won’t be able to investigate this until the weekend. Happy to review a PR.

**Comment by jplock** [comment]
I just published `v0.10.0` to NPM
